Task_HelloWorld/index.html

```diff
@@ -6,6 +6,10 @@
   </head>
```

Comment:
There seems to be a typo in the heading tag. It should be 'Hello, World!', not 'Hello World:'